site/index.html

```diff
@@ -1,14 +1,102 @@
-    <h1>Homepage</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Welcome to Boston Arts Academy</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <div class="header-box">
+    <h1>Artist-scholars designing their own future.</h1>
+    <div>view their wokr</div>
+  </div>
+  <div class="upcoming events">
+    <h2>Upcoming events</h2>
+    <div>view all</div>
+    <div>
+      <div>
+        <div>March 10</div>
+        <div>Applications Are Due: Hoope to see you nect year!</div>
+      </div>
+      <div>
+        <div>March 17</div>
+        <div>The Art of Connection: A Virtual Symposium</div>
+      </div>
+      <div>
+        <div>June 23</div>
+        <div>Aretha Franklin Tribute: Gospel Years</div>
+      </div>
+      <div class="news">
+        <div>News</div>
+        <div>view All</div>
+      </div>
+      <div class="box 2" <h3>174 Ipswich Street</h3>
+        <div>Tour the New BAA</div>
+        <div> Professor Jonah Harding Wins Genius Grant</div>
+        <div>Alum Franny Hall on nine Years Leading a Non-Profit</div>
+        <h4>Student Spotlight</h4>
+        <div>Sandy Perdue</div>
+        <div>Class of '23 |</div>
+        <div>Music Department</div>
+        <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam tempus scelerisque orci eu rutrum. Nulla
+          blandit
+          pharetra urna ac fringilla. Nam mollis nunc sed turpis suscipit rutrum. Integer dui nisi, auctor at imperdiet
+          et,
+          consectetur vitae lacus. Ut nec aliquam felis, in consectetur est. Proin maximus quis odio et porta. Donec a
+          odio
+          fringilla, volutpat lectus mattis, blandit erat. Aenean dignissim dapibus nibh, ac malesuada risus gravida eu.
+          Ut
+          felis leo, accumsan vitae nibh ac, varius dignissim enim. Suspendisse pharetra cursus tortor, vel dignissim
+          purus
+          tincidunt a.</div>
+        <div>learn more</div>
+        <h5>Boston Arts Academy impact</h5>
+        <div>Nam urna mauris, viverra et sem ac, accumsan interdum tellus. Aenean facilisis erat eu quam scelerisque
+          tempus.
+          Duis cursus fermentum elit vel pulvinar. Sed in lectus tristique, aliquam arcu at, iaculis orci. Sed eu orci
+          sapien.
+          Nulla eget volutpat mauris. Phasellus semper sollicitudin dolor. Morbi ut interdum augue. Nam dignissim tempor
+          mattis. Proin sit amet mollis erat, nec viverra nulla. Mauris vehicula lectus non suscipit condimentum. Nulla
+          vestibulum consectetur pellentesque. Proin in pharetra eros. Aenean id suscipit dolor. Phasellus sed fermentum
+          nulla, id fermentum nisi.</div>
+        <div>View More Stats</div>
+        <circle 1></circle>
```

Comment:
`<circle 1>` will give you an issue. You probably want a div with a class instead, so like `<div class="circle-1">`? Also, try to give all elements that are the same the same class. So here you might just call it `<div class="impact-circle">` and you can reuse that for all the circles in this section.